src/App.js

```diff
@@ -1,9 +1,11 @@
-import React, { Component } from "react";
+import React from "react";
 import "./assets/css/App.scss";
+import Routes from "./routes";
 
 const App = () => (
```

Comment:
Thanks for the great work @Peace-Apple , However, the indenting here is causing issues with eslint,
You could write this block as 
```
const App = () => (
  <div>
    <Routes />
  </div>
);
```

Reply:
Okay @CryceTruly let me look into that

Reply:
@CryceTruly  I have fixed it now